packages/base-domains/base-domains.base/opam

```diff
@@ -0,0 +1,17 @@
+opam-version: "2.0"
+maintainer: "https://github.com/ocaml-multicore/multicore-opam/issues"
+description: """
+Domains-based parallelism distributed with the Multicore OCaml compiler"
+"""
+depends: [
+  "ocaml" {>="4.06.1"}
+  "ocaml-variants" {
+    = "4.12.0+domains+effects" |
+    = "4.12.0+domains" |
+    = "4.10.0+multicore" |
+    = "4.10.0+multicore+no-effect-syntax" |
+    = "4.06.1+multicore" |
+    = "4.04.2+multicore" |
+    = "4.02.2+multicore"
```

Comment:
I'll import these more historical compilers later; this PR only includes 4.12.0*